libs/remix-ui/workspace/src/lib/actions/index.ts

```diff
@@ -124,6 +124,14 @@ export const initWorkspace = (filePanelPlugin) => async (reducerDispatch: React.
           plugin.on('editor', 'editorMounted', async () => await plugin.fileManager.openFile(route))
         } else await basicWorkspaceInit(workspaces, workspaceProvider)
       } else await basicWorkspaceInit(workspaces, workspaceProvider)
+    } else if (localStorage.getItem("currentWorkspace")) {
+      const index = workspaces.indexOf(localStorage.getItem("currentWorkspace"))
```

Comment:
It may be better to store `localStorage.getItem("currentWorkspace")` in a variable